release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -849,6 +849,31 @@ Helm-based Operator enhancements include:
 {product-title} now properly supports the `terminationGracePeriodSeconds`
 parameter with the CRI-O container runtime.
 
+[discrete]
+[id="ocp-4-5-readyz-config"]
+==== /readyz configuration for API server health probe
+
+All {product-title} 4.5 clusters must be configured to use the `/readyz`
+endpoint for API server health checking to remain supported. Any clusters
```

Comment:
Does this happen automatically for IPI clusters? The intro makes it sound like an "all clusters" change, but the rest of the data makes it sound like a load balancer reconfiguration for UPI. Please consider clarifying the intro.

Reply:
Yeah, IPI has `/readyz` configured by default. I'll make it clearer this is a UPI config recommendation.